cli/volume.go

```diff
@@ -189,7 +192,6 @@ func (v *VolumeRemoveCommand) runVolumeRm(args []string) error {
 }
 
 // volumeRmExample shows examples in volume rm command, and is used in auto-generated cli docs.
-// TODO: add example
 func volumeRmExample() string {
-	return "Add volume create examples in code, not in doc."
+	return "$ pouch volume rm pouch-volume"
```

Comment:
Nothing will output?
I suggest if success, output the volume name to give a positive hint/

Reply:
yes, nothing will output when succeeded, I add "Deleted <volume name>"